modules/hcp-ibmz-adding-credentials-registry.adoc

```diff
@@ -0,0 +1,61 @@
+:_mod-docs-content-type: CONCEPT
+[id="hcp-ibmz-adding-credentials-registry_{context}"]
+= Adding Credentials and Registry Certificate Authority to the Management Cluster
+
+To pull the mirror registry images from the Management Cluster , you must first add Credentials and Registry Certificate Authority of mirror registry to the Management cluster. Follow the procedure below:
```

Comment:
```suggestion
To pull the mirror registry images from the management cluster, you must first add credentials and the registry certificate authority of the mirror registry to the management cluster.
```
Is the term "registry certificate authority of the mirror registry" redundant? Would it be accurate to simply write, "certificate authority of the mirror registry"?